cx_Freeze/freezer.py

```diff
@@ -267,45 +191,13 @@ def _FreezeExecutable(self, exe):
 
         # Search the C runtimes, using the directory of the python libraries
         # and the directories of the base executable
-        if sys.platform == "win32" and self.runtime_files:
-            search_dirs = set()
-            for filename in dependent_files:
-                search_dirs.add(os.path.split(filename)[0])
-            for filename in self.runtime_files:
-                for search_dir in search_dirs:
-                    filepath = os.path.join(search_dir, filename)
-                    if os.path.exists(filepath):
-                        dependent_files.add(filepath)
-
-        # Always copy the dynamic libraries into lib folder
-        target_dir = os.path.join(self.targetdir, "lib")
+        self._PlatformAddExtraExecutableDependencies(dependent_files=dependent_files)
+
         for source in dependent_files:
-            # but python dlls should be copied within the executable
-            if sys.platform == "win32" and source.endswith(python_libs):
-                target = os.path.join(self.targetdir, os.path.basename(source))
-            else:
-                target = os.path.join(target_dir, os.path.basename(source))
```

Comment:
I see some errors when the sample are run in a venv. This behavior isn't present in WinFreezer. A _CopyTopDependency should exist in it and take care of that.

Reply:
I think this behaviour was reproduced in WinFreezer.  Originally the line 
```
target = os.path.join(self._PlatformExecutableDependencyDir(), os.path.basename(source))
```
should have had the same effect, but the logic has now been moved into _CopyTopDependency() as you suggested below.